fastlane/docs/PluginsTroubleshooting.md

```diff
@@ -50,12 +50,12 @@ Your `Pluginfile` should look something like this
 ```ruby
 # Autogenerated by fastlane
 
-gem 'fastlane-plugin-xcversion'
+gem 'fastlane-plugin-ruby'
```

Comment:
@mfurtak Once we have more useful plugins we should add this here, since `ruby` is irrelevant for most iOS/Android devs